cflib/crypt/util.cpp

```diff
@@ -171,17 +172,19 @@ QByteArray x509CreateCertReq(const QByteArray & privateKey, const QList<QByteArr
 			extensions.add(new Cert_Extension::Subject_Alternative_Name(subjectAN));
 		}
 
+		std::vector<uint8_t> encoded;
+		DER_Encoder(encoded)
```

Comment:
Apparently, the way to go. Pass in the vector that gets populated instead of returning it.